flow/embedded_views.h

```diff
@@ -343,6 +343,23 @@ class EmbedderViewSlice {
     return DlRegion::MakeIntersection(getRegion(), DlRegion(query.roundOut()));
   }
 
+  // TODO(hellohuanlin): iOS only for now, but we should try it on other
+  // platforms. We should deprecate the above `region` function if we migrate
+  // all platforms to use this function.
+  DlRegion roundedInRegion(const SkRect& query) const {
```

Comment:
Perhaps have the region() method take an SkIRect and let the caller decide how to turn their SkRect into the appropriate pixel query?

Reply:
This should be a common functionality across platforms. I'd prefer to keep the logic here so we can try it on other platforms, rather than duplicate the logic.

Reply:
I guess my thinking was that this class should not be imposing rounding logic assumptions on the callers. It has "pixel based" data to share and should expect "pixel based" queries from the callers.

But this is just a nit suggestion...

Reply:
Gotcha. I do see pros and cons. Worth to think about.